server/src/main/java/io/crate/types/StringType.java

```diff
@@ -352,4 +352,11 @@ public Integer characterMaximumLength() {
     public long valueBytes(String value) {
         return RamUsageEstimator.sizeOf(value);
     }
+
+    @Override
+    public void addMappingOptions(Map<String, Object> mapping) {
+        if (!unbound()) {
```

Comment:
```suggestion
        if (unbound() == false) {
```
Not sure if want this consisntently in the codebase though

Reply:
We don't have any rule in regards to this